src/MongoDB/Exception/WriteException.c

```diff
@@ -41,7 +41,7 @@ static PHP_METHOD(WriteException, getWriteResult)
 #if PHP_VERSION_ID >= 70000
 	writeresult = zend_read_property(php_phongo_writeexception_ce, getThis(), ZEND_STRL("writeResult"), 0, &rv TSRMLS_CC);
 #else
-	writeresult = zend_read_property(php_phongo_writeexception_ce, getThis(), ZEND_STRL("writeResult"), 0 TSRMLS_CC);
+	writeresult                  = zend_read_property(php_phongo_writeexception_ce, getThis(), ZEND_STRL("writeResult"), 0 TSRMLS_CC);
```

Comment:
I don't understand why it made this change.

Reply:
Odd, but I gather it's related to the odd indent we saw earlier.